File: ddpoliglot_v6_flutter/lib/models/data/SpeechFile.dart
// ignore_for_file: file_names, hash_and_equals, annotate_overrides
import 'package:json_annotation/json_annotation.dart';

part 'SpeechFile.g.dart';

@JsonSerializable()
class SpeechFile {
  int speechFileID;
  int hashCode;
  String speechFileName;
  int duration;
  int version;

  SpeechFile(
    this.speechFileID,
    this.hashCode,
    this.speechFileName,
    this.duration,
    this.version,
  );

  factory SpeechFile.fromJson(Map<String, dynamic> data) =>
      _$SpeechFileFromJson(data);

  Map<String, dynamic> toJson() => _$SpeechFileToJson(this);
}

File: ddPoliglotV6.Data/Models/SpeechFile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ddPoliglotV6.Data.Models
{
    public class SpeechFile
    {
        public int SpeechFileID { get; set; }
        public Int64 HashCode { get; set; }
        public string SpeechFileName { get; set; }
        public int Duration { get; set; }
        public int Version { get; set; }
    }
}
